test/integration/short/client-batch-tests.js

```diff
@@ -11,7 +11,7 @@ var vit = helper.vit;
 
 describe('Client', function () {
   this.timeout(120000);
-  describe('#batch(queries, {prepare: 0}, callback)', function () {
+  describe('#batch(queries, {prepare: 0}, callback) |smoke|', function () {
```

Comment:
Unsure if this is the best solution (adding `|smoke|` to the description), but seems to work.